src/routes/github/github-router.ts

```diff
@@ -48,26 +48,22 @@ subRouter.use(GithubOAuthRouter);
 subRouter.use(jiraSymmetricJwtMiddleware);
 subRouter.use(GithubServerAppMiddleware);
 
-// OAuth Routes
-subRouter.use(GithubOAuthRouter);
-
 // CSRF Protection Middleware for all following routes
 subRouter.use(csrfMiddleware);
 
 subRouter.use("/setup", GithubSetupRouter);
 
-// TODO: use GithubAuthMiddleware for all paths below once "create branch" supports JWT
-// subRouter.use(GithubAuthMiddleware);
-
 // App Manifest flow routes
 subRouter.use("/manifest", GithubManifestRouter);
 
-subRouter.use("/configuration", GithubAuthMiddleware, GithubConfigurationRouter);
+subRouter.use(GithubServerAppMiddleware);
```

Comment:
LOL, it is `GithubAuthMiddleware` instead of `GithubServerAppMiddleware`

Reply:
oh dear